### PR TITLE
modify eventhub reademe

### PR DESCRIPTION
**Pull request by yiliuTo**
modify README files of spring cloud eventhub modules for the release

### CODE REVIEW

File: sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ logging.level.org.springframework.web=DEBUG
 logging.level.org.hibernate=ERROR
 ```
 
-For more information about setting loging in pring, please refer to the [official doc](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-logging).
+For more information about setting loging in spring, please refer to the [official doc](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-logging).
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-starter-active-directory-b2c/README.md:223]: typo in loging

File: sdk/spring/azure-spring-cloud-eventhubs-stream-binder/README.md [hunk 2]
```diff
@@ -3,6 +3,25 @@
 The project provides **Spring Cloud Stream Binder for Azure Event Hub** which allows you to build message-driven 
 microservice using **Spring Cloud Stream** based on [Azure Event Hub](https://azure.microsoft.com/en-us/services/event-hubs/) service.
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-eventhubs-stream-binder/README.md:4]: Following the [Link Guidelines](https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md#link-guidelines) we should remove this `en-us` locale.

Comment (saragluna) [reply sdk/spring/azure-spring-cloud-eventhubs-stream-binder/README.md:4]: How about we put these links URL at the bottom of the file using reference links?

File: sdk/spring/azure-spring-cloud-eventhubs-stream-binder/README.md [hunk 3]
```diff
@@ -154,5 +171,39 @@ you can handle the error message in this way:
...
+
+For more information about setting logging in spring, please refer to the [official doc](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-logging).
+ 
+
 ## Next steps
+
+The following section provide a sample project illustrating how to use the starter.
+### More sample code
+- [Eventhubs Binder Sample](../azure-spring-boot-samples/azure-spring-cloud-eventhubs-binder-sample)
+- [Eventhubs Multibinders Sample](../azure-spring-boot-samples/azure-spring-cloud-eventhubs-multibinders-sample)
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-eventhubs-stream-binder/README.md:196]: same here

File: sdk/spring/azure-spring-cloud-eventhubs-stream-binder/README.md [hunk 4]
```diff
@@ -154,5 +171,39 @@ you can handle the error message in this way:
...
+
 ## Contributing
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+Please follow [instructions here](../CONTRIBUTING.md) to build from source or contribute.
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-eventhubs-stream-binder/README.md:201]: same here.

File: sdk/spring/azure-spring-cloud-starter-eventhubs-kafka/README.md [hunk 5]
```diff
@@ -1,8 +1,65 @@
...
+```
+
+For more information about setting logging in spring, please refer to the [official doc](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-logging).
+ 
+
 ## Next steps
+
+The following section provide a sample project illustrating how to use the starter.
+### More sample code
+- [Eventhubs Kafka Sample](../azure-spring-boot-samples/azure-spring-cloud-eventhubs-kafka-sample)
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-eventhubs-kafka/README.md:51]: relative URL

File: sdk/spring/azure-spring-cloud-starter-eventhubs-kafka/README.md [hunk 6]
```diff
@@ -1,8 +1,65 @@
...
+
 ## Contributing
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+Please follow [instructions here](../CONTRIBUTING.md) to build from source or contribute.
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-eventhubs-kafka/README.md:56]: relative URL

File: sdk/spring/azure-spring-cloud-starter-eventhubs/README.md [hunk 7]
```diff
@@ -1,8 +1,67 @@
...
 ## Next steps
+
+The following section provide a sample project illustrating how to use the starter.
+### More sample code
+- [Eventhubs Integration Sample](../azure-spring-boot-samples/azure-spring-cloud-eventhubs-integration-sample)
+
 ## Contributing
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+Please follow [instructions here](../CONTRIBUTING.md) to build from source or contribute.
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-eventhubs/README.md:59]: relative URL.

File: sdk/spring/azure-spring-cloud-starter-eventhubs/README.md [hunk 8]
```diff
@@ -1,8 +1,67 @@
...
+```
+
+For more information about setting logging in spring, please refer to the [official doc](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-logging).
+ 
+
 ## Next steps
+
+The following section provide a sample project illustrating how to use the starter.
+### More sample code
+- [Eventhubs Integration Sample](../azure-spring-boot-samples/azure-spring-cloud-eventhubs-integration-sample)
```
Comment (saragluna) [inline sdk/spring/azure-spring-cloud-starter-eventhubs/README.md:54]: relative URL.

File: sdk/spring/azure-spring-integration-eventhubs/README.md [hunk 9]
```diff
@@ -1,9 +1,66 @@
...
+```
+
+For more information about setting logging in spring, please refer to the [official doc](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-logging).
+ 
+
 ## Next steps
+
+The following section provide a sample project illustrating how to use this package.
+### More sample code
+- [Eventhubs Integration Sample](../azure-spring-boot-samples/azure-spring-cloud-eventhubs-integration-sample)
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-eventhubs/README.md:53]: relative URL.

File: sdk/spring/azure-spring-integration-eventhubs/README.md [hunk 10]
```diff
@@ -1,9 +1,66 @@
...
+
 ## Contributing
+This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+Please follow [instructions here](../CONTRIBUTING.md) to build from source or contribute.
```
Comment (saragluna) [inline sdk/spring/azure-spring-integration-eventhubs/README.md:58]: relative URL.

**Review by saragluna — CHANGES_REQUESTED** [review]
Generally great, but if we follow the [Link Guidelines](https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md#link-guidelines):
- No locale info should be contained in the URL.
- We should not use relative URLs.

And I suggest that we put the URLs together at the bottom of README file which will make the content short and easy to edit.

**Comment by yiliuTo** [comment]
@saragluna If we seperate readme modification into multiple PRs, then will there be conflicts when we merge each of them?

**Comment by saragluna** [comment]
If each PR modifies a subset of README files then it should be fine.

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**